### PR TITLE
Add an alternative way to install the client

### PR DESCRIPTION
**Pull request by burgyl**
It didn't work for me to install it in ~/bin, so I installed it in /usr/bin.
I added a short description to install it like I did.

### CODE REVIEW

**Comment by raylee** [comment]
Hello, and thanks for the report! If it didn't work for you in ~/bin then the docs definitely need to be updated. Can you do me a favor and do a fresh login on your system, open a shell, and do

`echo $PATH`

and check for ~/bin in the list somewhere? The readme recommends adding that, but I think it the installation section should mention that you may need to log out and back in again for the changes to $PATH to take effect.

**Comment by burgyl** [comment]
Hello, yes, I do have ~/bin in my path. I did log out for the changes to take effect. I should tell you that I'm running on wsl with zsh. I also recheck the rights on file and they are right.

**Comment by raylee** [comment]
Installing tldr in ~/bin is how I run it everywhere, so it's definitely a supported configuration. I'll do a bit of reading on the windows subsystem for linux (or install it in a VM) and check what the issue is.

In the meantime, if you have a bit of time for debugging the root cause:

`echo ~/bin` from the zsh in WSL,
add a short shell script in ~/bin, something like:
```
#!/bin/sh
echo Hello world
```
chmod +x the new script, and see if it runs? If my guess is right it won't, but this would definitely help narrow down where the issue is.

**Comment by burgyl** [comment]
Thank you for looking at this.

Yes, it won't. If you want, here are the results of the commands :

```echo ~/bin```
```
/home/guduche/bin
```

```ls -lah /home/guduche/bin```
```
total 12K
drwxrwxrwx 1 guduche guduche 4.0K Dec  3 09:41 .
drwxr-xr-x 1 guduche guduche 4.0K Dec  3 09:42 ..
-rwxrwxrwx 1 guduche guduche 8.7K Dec  2 21:28 tldr
-rwxrwxrwx 1 guduche guduche   28 Dec  3 09:40 tldrtest
```

```tldrtest```
```
zsh: command not found: tldrtest
```

**Comment by burgyl** [comment]
I think I'm doing something wrong, because I can't install it on a raspbian instance or on an ubuntu server vm.
I can run it with the complete path, but I can't only with ```tldr``` even when I have the folder in my path variable. (```…:~/bin```)

Don't look into the WSL to deeply, because the problem shouldn't come from there. Don't you think ?

**Comment by raylee** [comment]
Yes, I see your point. Could you try this please? Any of the systems will do, but I only checked this under zsh (with tldr in ~/bin and nowhere else):
```
PATH=$PATH:~/bin
which tldr
tldr
```

**Comment by burgyl** [comment]
Yes, it works thank you very much !
I see the problem, now the path to the bin is fully in the ```$PATH```, not the ```~```.
I've put it in my rc file and it's works perfectly.

**Comment by raylee** [comment]
Glad it's working :-)
